libraries/AP_HAL_ChibiOS/hwdef/podmanF405/hwdef-bl.dat

```diff
@@ -0,0 +1,39 @@
+# hw definition file for processing by chibios_pins.py
+# for minimal F405 bootloader
+
+# MCU class and specific type
+MCU STM32F4xx STM32F405xx
+
+# board ID for firmware load
+APJ_BOARD_ID 876

```

Comment:
These boards have to be reserved with an entry in `Tools/AP_Bootloader/board_types.txt`.

Please make a separatre PR to reserve those IDs, then use those IDs in these files.  Note the ID should be the net available from the ArduPilot rnage, which starts at 1000 (so roughly 1115)

Reply:
> These boards have to be reserved with an entry in `Tools/AP_Bootloader/board_types.txt`.
> 
> Please make a separatre PR to reserve those IDs, then use those IDs in these files. Note the ID should be the net available from the ArduPilot rnage, which starts at 1000 (so roughly 1115)

I closed this Pull request.
changed into [#24161](https://github.com/ArduPilot/ardupilot/pull/24161)